### PR TITLE
MOB-2516 [Publish Release 1.0.1]

### PR DESCRIPTION
**Pull request by JeevanThandi**

### CODE REVIEW

File: KarhooSDK.podspec [hunk 1]
```diff
@@ -1,19 +1,19 @@
 Pod::Spec.new do |s|
 
   s.name                  = "KarhooSDK"
-  s.version               = "1.0.0"
+  s.version               = "1.0.1"
   s.summary               = "Karhoo Network SDK"
   s.homepage              = "https://docs.stg.karhoo.net/v1/mobilesdk/network"
-  s.license               = 'MIT'
+  s.license               = 'BDS-2'
```
Comment (mohadian) [inline KarhooSDK.podspec]: I think it should be `BSD-2-Clause`

File: KarhooSDK/1.0.1/KarhooSDK.podspec [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Pod::Spec.new do |s|
+
+  s.name                  = "KarhooSDK"
+  s.version               = "1.0.1"
+  s.summary               = "Karhoo Network SDK"
+  s.homepage              = "https://docs.stg.karhoo.net/v1/mobilesdk/network"
+  s.license               = 'BDS-2'
+  s.author                = { "Karhoo" => "ios@karhoo.com" }
```
Comment (mohadian) [inline KarhooSDK/1.0.1/KarhooSDK.podspec:8]: Is the email correct? Do we check this email?

**Review by mohadian — CHANGES_REQUESTED** [review]

**Review by Tbruni85 — DISMISSED** [review]

**Review by mohadian — APPROVED** [review]

**Pull request merged**